src/foam/u2/view/UnstyledTableView.js

```diff
@@ -405,10 +405,16 @@ foam.CLASS({
          */
           var view = this;
 
-          var modelActions = view.of.getAxiomsByClass(foam.core.Action);
-          var actions = Array.isArray(view.contextMenuActions)
-            ? view.contextMenuActions.concat(modelActions)
-            : modelActions;
+          var actions = {};
+          var actionMapper = function(ret, action) {
+            ret[action.name] = action;
+            return ret;
+          };
+
+          // Model actions
+          view.of.getAxiomsByClass(foam.core.Action).reduce(actionMapper, actions);
```

Comment:
Why using reduce()?  Wouldn't forEach() be simpler since you don't actually replace 'actions'?

Reply:
You're right. Fixed in https://github.com/foam-framework/foam2/pull/4206/commits/103c41c008c4a2a1215aea6603b24981e0313f7c.